pydra/engine/task.py

```diff
@@ -348,10 +346,37 @@ def _command_args_single(self, state_ind, ind=None):
                 cpath = cdir.joinpath(lpath.name)
                 value = str(cpath)
             if f.type is bool:
+                if "argstr" in f.metadata:
+                    cmd_add.append(f.metadata["argstr"])
                 if value is not True:
                     break
             else:
-                cmd_add += ensure_list(value, tuple2list=True)
+                if "argstr" in f.metadata:
+                    argstr = f.metadata["argstr"]
+
+                    if argstr.endswith("..."):
+                        argstr = argstr.replace("...", "")
+                        if "sep" in f.metadata and isinstance(value, list):
```

Comment:
this is the kind of place where InputMultiObject would be used and therefore value should always be a list.

Reply:
but I should be convert to list only if type is ` InputMultiObject`, or just convert everyone?

Reply:
if input type is InputMultiObject in nipype1 this value is always a list, so you don't have to check. this is the kind of check that can happen during annotation. when the instance sees an input with argstr.endswith('...') it can check if the input type is a list or an inputmultiobject.